hasura/migrations/default/1646250716299_create_table_public_vaults/up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE "public"."vaults"
+(
+  "id" bigserial NOT NULL,
+  "token_address" varchar,
+  "simple_token_address" varchar,
+  "type" integer NOT NULL,
+  "org_id" bigint NOT NULL,
+  "decimals" integer NOT NULL,
+  "vault_owner" bigint NOT NULL,
```

Comment:
```suggestion
  "owner_id" bigint NOT NULL,
```

as a visual reminder that it's a foreign key... or `owned_by`? i'm ok with either convention